src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,56 @@
+using Flux
+
+export googlenet
+
+function conv_block(kernelsize::Tuple{Int64,Int64}, inplanes::Int64, outplanes::Int64; stride::Int64=1, pad::Int64=0)
+  conv_layer = []
+  push!(conv_layer, Conv(kernelsize, inplanes => outplanes, stride = stride, pad = pad))
+  push!(conv_layer, BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inceptionblock(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5, pool_proj)
+  branch1 = conv_block((1,1), inplanes, out_1x1)
+
+  branch2 = Chain(conv_block((1,1), inplanes, red_3x3),
+             conv_block((3,3), red_3x3, out_3x3; pad=1))        
+
+  branch3 = Chain(conv_block((1,1), inplanes, red_5x5),
+             conv_block((5,5), red_5x5, out_5x5; pad=1)) 
+
+  branch4 = Chain(MaxPool((3, 3), stride=1, pad=1),
+             conv_block((1,1), inplanes, pool_proj))
+
+  inception_layer = x -> begin
+    y1 = branch1(x)
+    y2 = branch2(x)
+    y3 = branch3(x)
+    y4 = branch4(x)
```

Comment:
Here, ``branch1`` and on are Arrays and arrays are not callable. So, shall I change it to ``branch1...`` instead?

Reply:
I think converting ``conv_layer`` to a function will do.

Reply:
Not sure I totally follow your comment, but you should add `...` after `conv_block()` here as well to splat the array of layers into the `Chain`.

Reply:
Actually that would lead to call ``BatchNorm`` layer with input dimension x. So, I did the following instead. The error is ``dimension mismatch at dims=1( got 26 expected 28)``

Reply:
That will cause ``LoadError : syntax: "..." expression outside call``